### PR TITLE
Fixes #20890 - changes SSH key validation messages

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Comment by tstrachota** [comment]
:+1: Tested on hammer and it works well:
```
> hammer user ssh-keys add --user john --name Key --key "wrong format" 
Could not add SSH Key:
  Key is not a valid public ssh key
  Key must be in OpenSSH public key format.
  Fingerprint could not be generated
  Length could not be calculated
```

**Review by boaz0 — APPROVED** [review]
:+1 SGTM

File: app/models/ssh_key.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ class SshKey < ApplicationRecord
   validates :key,
     :presence => true,
     :ssh_key => true,
-    :format => { with: /\A(ssh|ecdsa)-.*\Z/ }
+    :format => { with: /\A(ssh|ecdsa)-.*\Z/, message: N_('must be in OpenSSH public key format.') }
```
Comment (ares) [inline app/models/ssh_key.rb]: the dot should not be part of the message, see the screen below

**Review by ares — CHANGES_REQUESTED** [review]
in UI, the message is composed with other validation messages, so it does not work well with sentences

key

**Comment by shiramax** [comment]
@ares , Thanks , I removed the dot.

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Thanks @shiramax, merged as 4a96211d33c261655c887bbb6c787e38d883200a

**Pull request closed without merging**